app/views/admin/r_and_r_items/_form.html.erb_bak

```diff
@@ -0,0 +1,93 @@
+<%= simple_form_for(@admin_r_and_r_item) do |f| %>
```

Comment:
delete this file you didn't mean to commit it

Reply:
Done.